### DISCUSSION TITLE
How to add customized models to auto3dseg?

### DISCUSSION THREAD

**Discussion by dirtycomputer** [Project-MONAI/MONAI#6582] [General]
How to add customized models to auto3dseg?

**Comment by Nic-Ma** [comment]
Hi @dongyang0122 ,

Could you please help share some guide on this question?

Thanks in advance.

**Comment by dirtycomputer** [comment]
In this [tutorial link](https://github.com/Project-MONAI/tutorials/tree/main/auto3dseg)
The code below ⬇️ can use dint, swinunetr, segresnet and segresnet2d models.
`python -m monai.apps.auto3dseg AutoRunner run --input='./task.yaml'`
But I have some customized models, like modifying the segment model architecture. How can I add these customized model to auto3dseg. And I just run  `python -m monai.apps.auto3dseg AutoRunner run --input='./task.yaml'` , so that I can run all models automatically.
